### DISCUSSION TITLE
Accessibility of data visualization colors in bar chart

### DISCUSSION THREAD

**Discussion by bthomasstill** [twilio-labs/paste#2866] [General] [answered]
We're trying to use Paste's data visualization colors to make a stacked bar chart. My designer has flagged that the colors used may not have enough contrast to be visible for people with reduced vision/color blindness. What are Paste's views on this?

<img width="1067" alt="Screenshot 2022-12-07 at 11 44 49" src="https://user-images.githubusercontent.com/114398889/206171910-078050d1-1238-417a-967c-d0efe2592af1.png">

**Comment by gloriliale** [comment, marked as answer]
Hi @bthomasstill! 

Thanks for your question! There's information about the accessibility of our data visualization colors in [this portion of our docs](https://paste.twilio.design/foundations/data-visualization#categorical), specifically:

> The categorical palette is fully 3:1 contrast-accessible against our $color-background-body token and has >2:1 contrast between neighboring colors. Since the contrast between neighboring colors falls short of the [WCAG 2.1 AA requirement for non-text](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
 contrast, we will also include color-agnostic accessibility recommendations to assist with data read, which will be detailed in [additional accessibility for charts](https://paste.twilio.design/foundations/data-visualization#additional-accessibility-for-charts) section.
 
The page details a few suggestions that could help with accessibility for this bar chart:
- Starting at a different spot in our color sequence (starting at color 3, for example, and using the sequence from there)
- Using divider lines between the colors to enhance their contrast
- Using thicker bars to allow for more surface area of each color and more contrast

Let us know if this information helps!
